src/order/boolean_algebra.lean

```diff
@@ -436,7 +439,7 @@ instance boolean_algebra.to_bounded_order [h : boolean_algebra α] : bounded_ord
 def generalized_boolean_algebra.to_boolean_algebra [generalized_boolean_algebra α] [order_top α] :
   boolean_algebra α :=
 { compl := λ a, ⊤ \ a,
-  inf_compl_le_bot := λ _, disjoint_sdiff_self_right,
+  inf_compl_le_bot := λ _, disjoint_sdiff_self_right.le_bot,
```

Comment:
You should consider changing the `inf_compl_le_bot` to use `disjoint`. I meant to do it but this wasn't pressing so long as it was defeq.

Reply:
That would probably make sense to do as a separate PR; are you happy to do it?

Reply:
This PR is already huge, so I definitely don't want to change more things in this PR!